index.html

```diff
@@ -232,7 +232,7 @@ <h2>Skills</h2>
       <input type="text" id="name" name="user_name" placeholder="Full name" maxlength='30' required />
       <input type="email" name="user_email" id="email" placeholder="Email address" required />
       <textarea name="comment" id="comment" placeholder='Enter text here' maxlength='500' required></textarea>
-      <button type="submit">Get in Touch</button>
+      <button type="submit" class="my-button" disabled>Get in Touch</button>
     </form>
```

Comment:
- [ ] The previous reviewer did not ask that you disable the button. The reviewer asked that you have style declarations for your `hover`, `active` and `disable` states for your buttons. Kindly remove the `disabled` attribute you added to this button. I can see that you have the declarations for the `hover` and `active` states. However, the style declaration for the `hover` is missing the `rgba` keyword. Kindly include it on line 240 of your stylesheet; `box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);`

Also, kindly add the `disabled` style declaration for the button. You can read this [stack overflow](https://stackoverflow.com/questions/14750078/style-disabled-button-with-css) article to see how to include the disabled style declaration for buttons 👍 💻